### ISSUE TITLE
Gossip_Scripts

### ISSUE THREAD

**Issue by LordPsyan** [mangosArchives/Mangos-One-server-old#29]
I don't know where the main website or forums is for mangos-one. but I have run into a major problem... I cannot get any gossip_scripts to work. They just don't show up on the npc. I even downloaded that jukebox script thingy, imported it and nothing. the npt_text comes up just fine. here is one of my scripts, if this helps any...

This is supposed to give an option to "help" and it spawns a small child. the option never shows up on the npc, and I cannot figure out why. This might be core related, which is why I posted the issue here.

http://pastebin.com/ETcXhEh9

**Comment by LordPsyan** [comment]
data_flags in gossip_scripts got me. udb wiki says it needs to be 0x01 for active. so I set it to 1. but... setting it to 0 finally brought up the option. temp_summon_creature (command 10) doesn't spawn. I duplicated another entry (with different coordinates, of course, and entry number) and it still doesn't spawn the npc. maybe I am doing something wrong, but there is very limited documentation on gossip stuffs, and what is out there is outdated\limited and extremely hard to understand.

so what's the website to get info on mangos-one? I know getmangos.com is down, and most likely will be forever... so how does one get support? IRC?
